backend/tests/test_rocm_download.py

```
@@ -0,0 +1,203 @@
+    Verify that download_rocm_binary():
+    1. Downloads the server archive and ROCm libs archive.
+    2. Extracts them into the backends/rocm directory.
+    3. Reports progress via the progress_manager.
+    """
+    import hashlib
+
+    server_sha = hashlib.sha256(fake_tar_gz).hexdigest()
+    libs_sha = hashlib.sha256(fake_tar_gz).hexdigest()
+
+    responses = {
+        f"https://github.com/jamiepine/voicebox/releases/download/v0.2.3/voicebox-server-rocm.tar.gz": FakeResponse(
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)